src/html_files/utils.ts

```diff
@@ -171,3 +173,37 @@ function allRunCPUListUnchanged(cpu_list) {
     }
     return true;
 }
+
+function get_inside_value(data) {
+    if ('F64' in data) {
+        return data['F64'] ?? undefined;
+    } else if ('UInt64' in data) {
+        return data['UInt64'] ?? undefined;
+    } else if ('String' in data) {
+        return data['String'] ?? undefined;
+    } else if ('Stats' in data) {
+        if (comparator == 'mean') {
+            return data['Stats']['mean'] ?? undefined;
```

Comment:
I think you can simply do `return data['Stats'][comparator] ?? undefined`

Reply:
Ah. Got it. Changing.